src/native/managed/compile-native.proj

```diff
@@ -18,6 +18,9 @@
         <SysRoot Condition="'$(CrossBuild)' == 'true' and '$(HostOS)' != 'windows'">$(ROOTFS_DIR)</SysRoot>
         <LinkerFlavor Condition="'$(CrossBuild)' == 'true' and '$(TargetsLinux)' == 'true'">lld</LinkerFlavor>
         <CustomLinkerArgToolchainArg Condition="'$(CrossBuild)' == 'true' and '$(HostArchitecture)' == '$(TargetArchitecture)' and '$(HostOS)' != 'windows'">--gcc-toolchain=$(ROOTFS_DIR)/usr</CustomLinkerArgToolchainArg>
+
+        <!-- disable on linux-bionic, for now -->
+        <UseNativeAotForComponents Condition="'$(TargetsLinuxBionic)' == 'true'">false</UseNativeAotForComponents>
```

Comment:
Please move it next to https://github.com/dotnet/runtime/blob/5508f791239fd51b1624cc50d7c96e395a7d9fd5/eng/Subsets.props#L128

The whole point was to control it centrally.

Reply:
It's a local workaround for a problem in this project file, as opposed to global statement.

Publishing for bionic is supposed to be supported for components but this project needs to configure it, like it's doing in the lines above for other crossbuilds (at minimum, set PublishAotUsingRuntimePack). I could have tried to fix it but official builds are failing and it was blocked before #103273 so I don't feel bad just blocking it "for now".

I'll move it so it doesn't conflict with #103375 but it will be in the wrong spot now given the state of main and I haven't made my mind if it will be in wrong spot after that PR still (the workaround is too detached from where the problem is).